dataland-frontend/src/router/index.ts

```diff
@@ -89,6 +91,12 @@ const routes = [
     name: "Company EU Taxonomy for specific reporting period",
     component: ViewFrameworkData,
   },
+  {
+    path: `/qualityassurance`,
+    props: true,
```

Comment:
In the Vue Router, the `props: true` option allows passing route parameters as props to the component rendered for the specified route. When you set `props: true` for a route, it automatically maps the route parameters to props with matching names in the component.

In this case, the path does not have any potential parameters. So why is `props: true` ?

Reply:
This seems like it was just copied. I remove the unnecessary part

Reply:
👍